browser/js/items/item_page.html

```diff
@@ -35,6 +38,11 @@
 					</div>
 				</div>
 			</div>
+			<div class="row">
```

Comment:
this row should be moved out so that it is outside the parent row. So the reviews would fall below the item picture and take the full width.